demo/horizontal/emmo2meta.py

```diff
@@ -29,7 +29,7 @@
 if sys.version_info >= (3, 7):
     odict = dict
 else:
-    from collections import OrderedDict as odict  # noqa: F401
+    from collections.abc import OrderedDict as odict  # noqa: F401
```

Comment:
I do not think OrderedDict is an abstract class? 

From the documentation here it is in collections.abc: https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes

However, the documentation [here](https://docs.python.org/3/library/collections.html) says that it will be moved. If so, that should apply for defaultdict as well. 

In my python3.8 importing OrderedDict from collections.abc does not work.

Reply:
Indeed, you're right. Also, as this will only be imported in Python versions lower than 3.7, and this package only supports 3.6 and 3.7, this is only relevant for 3.6, which indeed imports from `collections` straight up 👍